buildSrc/src/main/java/com/osacky/flank/gradle/validation/Util.kt

```diff
@@ -0,0 +1,17 @@
+package com.osacky.flank.gradle.validation
+
+import com.osacky.flank.gradle.FladleConfig
+import org.gradle.api.provider.ListProperty
+import org.gradle.api.provider.MapProperty
+import org.gradle.api.provider.Property
+import kotlin.reflect.full.memberProperties
+
+internal fun FladleConfig.getPresentProperties() = this::class.memberProperties
```

Comment:
in general, I don't like `Util` file name, its a bit generic. like _anything_ can go in there. I think this method should just go on the `FladleConfig` class itself.

Reply:
Makes sense, I'll move it 👍